data/campaigns/Wings_of_Victory/scenarios/01_The_Hunt.cfg

```diff
@@ -0,0 +1,450 @@
+        # Lynxes
+        {SCATTER_UNITS 3 "Lynx" 1 (
+            {EVERYWHERE}
+            terrain=*^F*
+            {SCATTER_NOT_NEXT}
+        ) (side=3)}
+        [micro_ai]
+            side=3
+            ai_type=big_animals
+            action=add
+            [filter]
+                type=Lynx
+            [/filter]
+            [filter_location]
+                terrain=*^F*,Gg
+            [/filter_location]
+            [filter_location_wander]
+                terrain=*^F*,Gg
+            [/filter_location_wander]
+            [avoid_unit]
+                [not]
+                    type=Rabbit
+                [/not]
+            [/avoid_unit]
+        [/micro_ai]
+        # Rabbit burrows
+        {PLACE_IMAGE scenery/burrow.png 25 37}
+        {PLACE_IMAGE scenery/burrow.png 34 33}
+        {PLACE_IMAGE scenery/burrow.png 40 23}
+        {PLACE_IMAGE scenery/burrow.png 41 31}
+        {PLACE_IMAGE scenery/burrow.png 44 38}
+        {PLACE_IMAGE scenery/burrow.png 49 28}
+        {PLACE_IMAGE scenery/burrow.png 50 44}
+        {PLACE_IMAGE scenery/burrow.png 53 11}
+        # Boars & Rabbits
+        {SCATTER_UNITS 9 "Tusklet,Tusklet,Tusker,Tusker,Tusklet,Tusker" 1 (
+            {EVERYWHERE}
+            terrain=*^F*
+            {SCATTER_NOT_NEXT}
+        ) (side=4)}
+        {SCATTER_UNITS 5 "Rabbit" 1 (
+            {EVERYWHERE}
+            terrain=*^F*
+            {SCATTER_NOT_NEXT}
+        ) (side=4)}
+        [micro_ai]
+            side=4
+            ai_type=forest_animals
+            action=add
+            rabbit_type=Rabbit
+            tusker_type=Tusker,Gorer
+            tusklet_type=Tusklet
+            [filter_location]
+                terrain=*^F*,Gg
+            [/filter_location]
+            rabbit_number=5
+            rabbit_enemy_distance=3
+            ## rabbit_hole_img=scenery/burrow.png # commented out due to bug in forest_animals mai, can't use any other [item] unless I'm fine with rabbits coming out of them until it is fixed.
+            ## Can remove comments and activate for 1.15.0, leave in 1.14.x, as it was only fixed in 1.14.6
+        [/micro_ai]
+
+        {MODIFY_UNIT side=1 facing ne}
+        [objectives]
+            [objective]
+                description= _ "All enemies are defeated and Galun has reached level two."
+                condition=win
+            [/objective]
+            {OBJECTIVE_OR}
+            [objective]
+                description= _ "Turns run out and Galun has reached level two."
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Death of Galun"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Vank"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "You lose more than one member of your hunting party."
```

Comment:
The phrasing of this objective doesn't match that of the standard objectives. Something like "Loss of two members of your hunting party" would fit better.

The other two objectives below also suffer from this problem, but I'm not sure how they can be fixed, and I think they have other problems too, so for now I won't make any suggestions.

Reply:
Noted.